<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanningWindow</class>
 <widget class="QMainWindow" name="PlanningWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planning des séances</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressources.qrc">
    <normaloff>:/images/calendar-icon.png</normaloff>:/images/calendar-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="CalendarView" name="planningView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="planifyAction"/>
   <addaction name="editAction"/>
   <addaction name="removeAction"/>
   <addaction name="separator"/>
   <addaction name="dayViewAction"/>
   <addaction name="weekViewAction"/>
   <addaction name="monthViewAction"/>
   <addaction name="separator"/>
   <addaction name="backAction"/>
   <addaction name="todayAction"/>
   <addaction name="nextAction"/>
  </widget>
  <action name="planifyAction">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/appointment-new-icon.png</normaloff>:/images/appointment-new-icon.png</iconset>
   </property>
   <property name="text">
    <string>Planifier</string>
   </property>
   <property name="toolTip">
    <string>Planifier une séance</string>
   </property>
  </action>
  <action name="dayViewAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/calendar-day-icon.png</normaloff>:/images/calendar-day-icon.png</iconset>
   </property>
   <property name="text">
    <string>Vue Jour</string>
   </property>
   <property name="toolTip">
    <string>Vue au jour</string>
   </property>
  </action>
  <action name="weekViewAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/calendar-week-icon.png</normaloff>:/images/calendar-week-icon.png</iconset>
   </property>
   <property name="text">
    <string>Vue Semaine</string>
   </property>
   <property name="toolTip">
    <string>Vue à la semaine</string>
   </property>
  </action>
  <action name="monthViewAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/calendar-month-icon.png</normaloff>:/images/calendar-month-icon.png</iconset>
   </property>
   <property name="text">
    <string>Vue Mois</string>
   </property>
   <property name="toolTip">
    <string>Vue au mois</string>
   </property>
  </action>
  <action name="backAction">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/back-icon.png</normaloff>:/images/back-icon.png</iconset>
   </property>
   <property name="text">
    <string>Précédent</string>
   </property>
   <property name="toolTip">
    <string>Précédent</string>
   </property>
  </action>
  <action name="nextAction">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/next-icon.png</normaloff>:/images/next-icon.png</iconset>
   </property>
   <property name="text">
    <string>Suivant</string>
   </property>
   <property name="toolTip">
    <string>Suivant</string>
   </property>
  </action>
  <action name="removeAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/event-remove.png</normaloff>:/images/event-remove.png</iconset>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
   <property name="toolTip">
    <string>Supprimer une séance planifiée</string>
   </property>
  </action>
  <action name="editAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/edit-icon.png</normaloff>:/images/edit-icon.png</iconset>
   </property>
   <property name="text">
    <string>Editer</string>
   </property>
   <property name="toolTip">
    <string>Editer</string>
   </property>
  </action>
  <action name="todayAction">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/today-icon.png</normaloff>:/images/today-icon.png</iconset>
   </property>
   <property name="text">
    <string>Aujourd'hui</string>
   </property>
   <property name="toolTip">
    <string>Aller à aujourd'hui</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CalendarView</class>
   <extends>QGraphicsView</extends>
   <header>views/calendar/calendarview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
